payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/PaymentServiceImpl.java

```diff
@@ -33,6 +44,20 @@ public WalletEntity makePayment(final WalletPayment walletPayment) {
         final var updatedToWalletBalance = toWallet.getBalance().add(walletPayment.amount());
         updateBalance(updatedToWalletBalance, toWallet);
 
+        final var notificationClient = NotificationClient.builder()
+                .amount(walletPayment.amount())
+                .build();
+
+        final WebClient webClient = webClientConfig.getWebClient();
+        executorService.execute(() -> webClient
+                .post()
+                .uri(webClientConfig.getBaseUrl() + NOTIFICATION_PATH)
+                .bodyValue(notificationClient)
+                .retrieve()
+                .toBodilessEntity()
+                .block()
```

Comment:
@Hoocki @kostya4201337 
Remember, It's antipattern to use `.block()` in asynchronous application (except tests) as it breaks asynchronous way of programming. But as we use synchronous approach in this application, it's totally fine.